File: doc/quote-azure-k8-backend/Models/Auth/LoginRequest.cs
using System.ComponentModel.DataAnnotations;

namespace quote_azure_k8_backend.Models.Auth
{
    public class LoginRequest
    {
        /// <summary>
        /// Email address or username for login
        /// </summary>
        [Required(ErrorMessage = "Email or username is required")]
        public string LoginIdentifier { get; set; } = string.Empty;
        
        [Required(ErrorMessage = "Password is required")]
        public string Password { get; set; } = string.Empty;
    }
}

File: quote-api/src/main/java/com/quote/k8/dto/LoginRequest.java
package com.quote.k8.dto;

import jakarta.validation.constraints.NotBlank;

public class LoginRequest {
    
    @NotBlank(message = "Email or username is required")
    public String loginIdentifier;
    
    @NotBlank(message = "Password is required")
    public String password;
}
